src/utils/test/validAttr.test.js

```diff
@@ -439,6 +439,7 @@ describe('validAttr', () => {
     expect(validAttr('onFocus')).toEqual(true)
     expect(validAttr('onFocusCapture')).toEqual(true)
     expect(validAttr('onBlur')).toEqual(true)
+    expect(validAttr('onInput')).toEqual(true)
```

Comment:
@kitten I actually added the wrong test anyways!! It were supposed to be onInvalid rather than onInput.. sorry for the confusion was way to early for me!